### PR TITLE
Remove standalone jobs

### LINKED ISSUE #1648: Remove standalone jobs
* They new payload concept couples the job more closely to the workflow instance. There is no simple way to maintain standalone jobs with payload.

### PR DESCRIPTION
**Pull request by ChrisKujawa**
closes #1648

### CODE REVIEW

File: clients/go/tests/integration/jobWorker_test.go [hunk 1]
```diff
@@ -27,11 +28,19 @@ var _ = Describe("JobWorker", func() {
...
+			jobs := jobStartCount
+
+			_, err := client.NewDeployWorkflowCommand().AddResourceFile("../../../java/src/test/resources/workflows/simple-process.bpmn").Send()
+			Expect(err).To(BeNil())
 
 			for i := 0; i < jobs; i++ {
 				go func() {
-					_, err := client.NewCreateJobCommand().JobType("foo").Retries(retries).Send()
+					_, err := client.
+                      NewCreateInstanceCommand().
```
Comment (Zelldon) [inline clients/go/tests/integration/jobWorker_test.go]: This indent is weird.In the ide it looks ok

Comment (Zelldon) [reply clients/go/tests/integration/jobWorker_test.go]: Ah the others are tabs instead of whitespaces :sweat_smile:

**Review by saig0 — APPROVED** [review]
bors r+

**Comment by saig0** [comment]
I like it :+1:

**Pull request merged**